.github/CODEOWNERS

```
@@ -13,7 +13,7 @@
 /shell.nix @NixOS/org
 
 /doc/org-repo.md @NixOS/steering
-/doc/discourse.md @NixOS/steering
+/doc/discourse.md @NixOS/steering @mweinelt @picnoir @lassulus
```

Because all Discourse admins can change who's an admin, all of them should be listed here and informed about it, to make sure they don't change administrators without a PR.

---

```suggestion
/doc/discourse.md @NixOS/steering @mweinelt @picnoir @lassulus @jtojnar @ctheune @dpausp
```

Also @jtojnar because not being part of the SC anymore doesn't automatically mean he won't be an admin anymore.

Ping @ctheune, @dpausp, @picnoir and @mweinelt